mmv1/products/networkconnectivity/RegionalEndpoint.yaml

```diff
@@ -0,0 +1,127 @@
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_regional_endpoint_regional_access'
+    primary_resource_id: 'default'
+    vars:
+      rep_name: 'my-rep'
+      rep_target_google_api: 'spanner.me-central2.p.rep.googleapis.com'
+      rep_access_type: 'REGIONAL'
+      rep_address: 'my-address'
+      rep_network: 'my-network'
+      rep_subnetwork: 'my-subnetwork'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_regional_endpoint_global_access'
+    primary_resource_id: 'default'
+    vars:
+      rep_name: 'my-rep'
+      rep_target_google_api: 'spanner.me-central2.p.rep.googleapis.com'
+      rep_access_type: 'GLOBAL'
+      rep_address: 'my-address'
+      rep_network: 'my-network'
+      rep_subnetwork: 'my-subnetwork'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of a RegionalEndpoint. Format: `projects/{project}/locations/{location}/regionalEndpoints/{regional_endpoint}`.
+
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Time when the RegionalEndpoint was created.
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Time when the RegionalEndpoint was updated.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      User-defined labels.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A description of this resource.
+  - !ruby/object:Api::Type::String
+    name: 'targetGoogleApi'
+    required: true
+    description: |
+      The service endpoint this private regional endpoint connects to. Format: `{apiname}.{region}.p.rep.googleapis.com` Example: \"cloudkms.us-central1.p.rep.googleapis.com\".
+  - !ruby/object:Api::Type::String
+    name: 'network'
+    description: |
+      The name of the VPC network for this private regional endpoint. Format: `projects/{project}/global/networks/{network}`
+  - !ruby/object:Api::Type::String
+    name: 'subnetwork'
+    description: |
+      The name of the subnetwork from which the IP address will be allocated. Format: `projects/{project}/regions/{region}/subnetworks/{subnetwork}`
+  - !ruby/object:Api::Type::Enum
+    name: accessType
+    required: true
+    description: |
+      The access type of this regional endpoint. This field is reflected in the PSC Forwarding Rule configuration to enable global access.
+    values:
+      - :ACCESS_TYPE_UNSPECIFIED
```

Comment:
This value is not needed, as it is meaningless.

Reply:
Done.